[2022-03-17 19:47:33,795] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:33,805] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:33,805] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:33,806] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-17 19:47:33,806] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:33,816] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): remove_files_task> on 2019-01-01 00:00:00+00:00
[2022-03-17 19:47:33,821] {standard_task_runner.py:52} INFO - Started process 613 to run task
[2022-03-17 19:47:33,826] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'remove_files_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmp5xto2phg', '--error-file', '/tmp/tmp6k689_fn']
[2022-03-17 19:47:33,827] {standard_task_runner.py:77} INFO - Job 23: Subtask remove_files_task
[2022-03-17 19:47:33,889] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 06aea6ade097
[2022-03-17 19:47:33,930] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-17 19:47:33,950] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=remove_files_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-03-17 19:47:33,951] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 19:47:33,952] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/dft-road-casualty-statistics-accident-2019.csv /opt/***/dft-road-casualty-statistics-accident-2019.parquet']
[2022-03-17 19:47:33,965] {subprocess.py:85} INFO - Output:
[2022-03-17 19:47:33,973] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-17 19:47:33,999] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=remove_files_task, execution_date=20190101T000000, start_date=20220317T194733, end_date=20220317T194733
[2022-03-17 19:47:34,040] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 19:47:34,072] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-20 06:18:15,349] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:18:15,366] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:18:15,366] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:18:15,367] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-20 06:18:15,367] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:18:15,384] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): remove_files_task> on 2019-01-01 00:00:00+00:00
[2022-03-20 06:18:15,391] {standard_task_runner.py:52} INFO - Started process 377 to run task
[2022-03-20 06:18:15,397] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'remove_files_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmpwms850_h', '--error-file', '/tmp/tmpce8qpk8n']
[2022-03-20 06:18:15,398] {standard_task_runner.py:77} INFO - Job 43: Subtask remove_files_task
[2022-03-20 06:18:15,488] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 974079c1cd0d
[2022-03-20 06:18:15,548] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-20 06:18:15,576] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=remove_files_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-03-20 06:18:15,577] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-20 06:18:15,578] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/dft-road-casualty-statistics-accident-2019.csv']
[2022-03-20 06:18:15,599] {subprocess.py:85} INFO - Output:
[2022-03-20 06:18:15,611] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-20 06:18:15,649] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=remove_files_task, execution_date=20190101T000000, start_date=20220320T061815, end_date=20220320T061815
[2022-03-20 06:18:15,691] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-20 06:18:15,748] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
